php_phongo.h

```diff
@@ -44,6 +44,7 @@ ZEND_BEGIN_MODULE_GLOBALS(mongodb)
 	char *debug;
 	FILE *debug_fd;
 	bson_mem_vtable_t bsonMemVTable;
+	HashTable clients;
```

Comment:
The [Internals Book](http://www.phpinternalsbook.com/hashtables/hashtable_api.html) talks about using `ALLOC_HASHTABLE()`, but that is only useful for non-persistent hash tables. Allocating the value inside of our global struct seemed kosher.